Doc updates related to changes to vic-machine create options

stuclem opened the pull request:
Fixes https://github.com/vmware/vic/issues/1284, https://github.com/vmware/vic/issues/1309, https://github.com/vmware/vic/issues/877, https://github.com/vmware/vic/issues/1008, https://github.com/vmware/vic/issues/871. 

Updated the descriptions of the vic-machine create options, the deployment instructions, examples.

andrewtchin commented:
lgtm

emlin commented on doc/user_doc/vic_installation/vch_installer_examples.md:
```diff
@@ -21,16 +21,15 @@ You can install vSphere Integrated Containers directly on an ESXi host that is n
 ## Deploy a Virtual Container Host in a vCenter Server Cluster ##
 
-If vCenter Server manages clusters, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
+If vCenter Server manages more than one cluster, you use the `compute-resource` option to specify the address of the cluster to which to deploy the virtual container host. You provide the cluster address in the `govc` format.
```
Actually, here we did enhancement to make it easier.
So relative path is accepted, for example, you can specify path as host/Resources/path/somewhere.
Or in ESXi, you could just write the path after Resources, for example path/somewhere
In VC, that could be 'vc cluster'/path/somewhere

emlin commented on doc/user_doc/vic_installation/vch_installer_options.md:
```diff
@@ -150,17 +213,26 @@ Use this option in combination with the `cert` option, that provides the path to
+
+Short name: `-f`
+
 Forces `vic-machine` to ignore warnings and non-fatal errors and continue with the deployment of a virtual container host. Errors such as an incorrect compute resource still cause the installation to fail. Using the `force` option deletes any existing virtual container host appliances or datastore folders that have the same name as the one you are specifying in the current deployment. 
```
The delete function of force install is removed, cause we already had vic-machine delete, that one is more clear and not easy to cause unexpected result.

stuclem replied:
Deleted the sentence about deleting an existing installation.

stuclem commented:
[skip ci]

stuclem commented:
@emlin I have updated the following topics to correct the paths for compute-resource:

https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/vch_installer_options.md
https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/vch_installer_examples.md

I also updated the confirmation message after a successful VCH deployment in https://github.com/stuclem/vic/blob/stuclem-tp3/doc/user_doc/vic_installation/install_vic_cli.md.

Is this OK now?
Thanks!

stuclem commented:
:recycle: just in case...

mdubya66 commented:
lgtm

stuclem commented:
:recycle:
